grails-app/controllers/org/pih/warehouse/api/CombinedShipmentItemApiController.groovy

```diff
@@ -176,7 +184,9 @@ class CombinedShipmentItemApiController {
             }
         }
 
-        response.setHeader("Content-disposition", "attachment; filename=\"Order-items-template.csv\"")
+        String filename = g.message(code: "orderItem.template.header.label", default: "Order-items-template")
```

Comment:
is this code correct?

Reply:
thanks for catching that - I forgot I included it also in the response header.